doc/release/1.16.0-notes.rst

```diff
@@ -34,6 +34,12 @@ New Features
 Improvements
 ============
 
+``randint`` and ``choice`` now work on empty distributions
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Even when no elements needed to be drawn, ``np.random.randint`` and
+``np.random.choice`` raised an error when the arguments described an empty
+distribution. This has been fixed so that e.g.
+``np.random.choice([],0) == np.array([],dtype=float64)``.
```

Comment:
nit: spaces after commas

Reply:
fixed, also replaced all spaces after comments in these three files `grep ",[^ ]"`